### DISCUSSION TITLE
auto increment id

### DISCUSSION THREAD

**Discussion by grafst** [spatie/laravel-prefixed-ids#19] [Q&A]
Great Package 
I love speaking IDs!

Is there a way to instead of uuids use the Laravel auto incrementing id?
or does that not work?
also it would be cool to pad the id with zeros.

example: 
model_id:3
prefixed_id:XY003

**Comment by ercsctt** [comment]
Auto incrementing IDs are handled by the database, not the application; so this wouldn't be possible without looking up the previous record and incrementing manually.
